styleguide/source/_patterns/01-molecules/06-components/22-article-preview/02-article-preview-small/article-preview-small.twig

```diff
@@ -0,0 +1,7 @@
+<div class="topic-articles_article-preview">
```

Comment:
The pattern in this PR is called "Topic Articles" (as in, articles that are part of the topic), so the class is named `.topic-articles_article-preview` because it's an article preview (which is what I'm calling the links to the articles) that's nested within the Topic Articles section.

The `.topic_article-preview` was the first "article preview" pattern I created, so I was thinking of it as the base molecule. I know it's inconsistent with `.topic-articles_article-preview` because the class isn't named for the section like it is there (if it were it would be `.topic-hero_article-preview`). Currently I'm styling the molecule that's nested in the Hero with `.topic-hero .topic_article-preview` because I didn't want to modify the original base molecule.

That being said, I'm getting confused because several of the patterns have what I'd call a `.topic_article-preview`, but I'm not sure if we actually want to reuse the base molecule because they can sometimes differ slightly. I'm not sure which way to go here, but that's the reasoning behind the inconsistency.